<ui version="4.0" >
 <class>VocableEdit</class>
 <widget class="QDialog" name="VocableEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VocableEditor</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="icons.qrc" >:/icons/icons/table_edit.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="5" column="0" >
        <widget class="QLabel" name="queryCountLabelLeft" >
         <property name="text" >
          <string>Query Count:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Lesson:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>lessonComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="VocableEditorTextEdit" name="foreignTextEdit" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="boxLabelLeft" >
         <property name="text" >
          <string>Box:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="lessonComboBox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="lastQueryLabelLeft" >
         <property name="text" >
          <string>Last Query:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLabel" name="lastQueryLabel" />
       </item>
       <item row="5" column="1" >
        <widget class="QLabel" name="queryCountLabel" />
       </item>
       <item row="3" column="1" >
        <widget class="QLabel" name="boxLabel" />
       </item>
       <item row="0" column="1" >
        <widget class="VocableEditorTextEdit" name="nativeTextEdit" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="foreignLabel" >
         <property name="text" >
          <string>&amp;Foreign:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>foreignTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="nativeLabel" >
         <property name="text" >
          <string>&amp;Native:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>nativeTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0" >
        <widget class="QLabel" name="badCountLabelLeft" >
         <property name="text" >
          <string>Bad Count:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1" >
        <widget class="QLabel" name="badCountLabel" />
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="translateGroupBox" >
      <property name="title" >
       <string>dictionary</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>6</number>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <property name="bottomMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="directoryComboBox" />
       </item>
       <item>
        <widget class="QTreeWidget" name="translateTreeWidget" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>English</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>German</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okAddNewButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VocableEditorTextEdit</class>
   <extends>QTextEdit</extends>
   <header>VocableEditorTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nativeTextEdit</tabstop>
  <tabstop>foreignTextEdit</tabstop>
  <tabstop>lessonComboBox</tabstop>
  <tabstop>okAddNewButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>VocableEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>543</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel" >
     <x>524</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>VocableEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>624</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel" >
     <x>579</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
